modules/ROOT/pages/mule-deployment-model.adoc

```diff
@@ -0,0 +1,26 @@
+= Mule App Deployment Model
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, deploy, architecture
+
+A Mule project for deployment is a Mule application. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
+
+* Running independently multiple applications side-by-side. If you have to build an application with some complexity, you can divide it by creating several apps with their specific logic and then deploy all the apps within the same Mule container.
```

Comment:
The first sentence repeats the lead-in to the list. You don't need it. Suggestion for the second sentence:

To build a complex app, you can break into several smaller apps and then deploy the apps into the same Mule container.

Also - I think we usually use the term app and not application.It should be consistent throughout the topic except maybe on first mention: A Mule project for deployment is a Mule application (Mule app).

Reply:
Removed repetitive first sentence and updated second sentence.
For this doc I would leave _application_ term since it references other inline links processes and all of them use the term "application/s"